src/test/java/com/cloudbees/jenkins/support/filter/ContentMappingsTest.java

```diff
@@ -95,7 +96,7 @@ public void contentMappingsOrderedByLengthDescending() throws IOException {
         List<String> originals = StreamSupport.stream(mappings.spliterator(), false)
                 .map(ContentMapping::getOriginal)
                 .collect(toList());
-        Assertions.assertThat(originals).containsExactly("LongestNameHere", "LongerName", "ShortName");
+        assertThat(originals, contains("LongestNameHere", "LongerName", "ShortName"));
```

Comment:
Then it should verify the size as well?

Reply:
The `contains` Hamcrest matcher already does check the size, see the [Javadoc](http://hamcrest.org/JavaHamcrest/javadoc/2.2/org/hamcrest/Matchers.html#contains-E...-). It's just named differently than the AssertJ matcher which was used before. The Hamcrest matcher `hasItems` is the one that only checks for the specified items without the size, see [Javadoc](http://hamcrest.org/JavaHamcrest/javadoc/2.2/org/hamcrest/Matchers.html#hasItems-T...-).

Reply:
Thanks for the clarification.